### PR TITLE
made calculator functional

### PR DESCRIPTION
**Pull request by Developer-Kingz**
- added calculator.js and operate.js files
- added event listners to make calculator functional

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,34 +1,152 @@
...
+            text="%"
+            className="inputs"
+            onClick={() => handleButtonClick('%')}
+          />
+          <Button
+            type="button"
+            text="/"
+            className="inputs-right"
+            onClick={() => handleButtonClick('/')}
+          />
```
Comment (CollinsTatang) [inline src/components/Calculator.js]: Great job so far @Developer-Kingz but you still have some little fix here.

- [ ] Please your app throws an error when the user tries to use the divisible operator. 
error

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Developer-Kingz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json
Comment (AbelG101) [inline package.json:1]: - [OPTIONAL] The previous reviewer forgot to mention this but I noticed that the branch name you used is not very descriptive. To ensure good development practices, we recommend using a branch name that clearly indicates the feature or action you're working on. For instance, `add_objects` would be a more descriptive and identifiable branch name. When working with GitHub, it's important to use descriptive branch names to help collaborators understand the context of your changes. Please consider using this approach or a similar one in the future to help streamline your development process and improve collaboration. Thank you! 🙂

**Review by AbelG101 — APPROVED** [review]
# Hi @Developer-Kingz 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- State is properly handled ✔
- Props are passed correctly to child components ✔
- Well-structured files ✔
- All linter checks are passing ✔
- Correct use of GitFlow ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**